### DISCUSSION TITLE
Icecast Failed to fetch mime-type

### DISCUSSION THREAD

**Discussion by lowepalma** [savonet/liquidsoap#3055] [User Support]
I use liquidsoap 2.1.4-win 64
I have tried using several Icecast streaming services that I add to my M3U playlist, but I always get this error. The stream plays, but there is no audio. I have also tried putting the URL in my script, but The same script works fine. The error message is: 

[procol.external:3] Failed to find a file extension for http://xxxx
[procol.external:3] Failed to fetch mime-type for http://xxxxxx

**Comment by toots** [comment]
Hi and thanks for reporting. Do you have an URL we could use to reproduce? You can send it privately at toots@rastageeks.org

**Comment by lowepalma** [comment]
The url is: https://eu1.lhdserver.es:2957/stream

**Comment by toots** [comment]
Ha I see now. Yeah, you can't use icecast urls in a playlist. They are infinite streams, they never end and so the download process would also never end. This is probablt what's happening, the initial mime fetch times out.
